diff --git a/app/models/status.rb b/app/models/status.rb
index 8bdb5e8db..879cf2f62 100644
--- a/app/models/status.rb
+++ b/app/models/status.rb
@@ -88,7 +88,7 @@ class Status < ApplicationRecord
 
   default_scope { recent.kept }
 
-  scope :recent, -> { reorder(id: :desc) }
+  scope :recent, -> { reorder(created_at: :desc) }
   scope :remote, -> { where(local: false).where.not(uri: nil) }
   scope :local,  -> { where(local: true).or(where(uri: nil)) }
   scope :with_accounts, ->(ids) { where(id: ids).includes(:account) }
