--- a/pyproject.toml	2026-08-03 22:22:11.492596363 +0100
+++ b/pyproject.toml	2026-08-03 22:22:29.621851337 +0100
@@ -1,9 +1,7 @@
 [tool.poetry]
 name = "graphql-core"
 version = "3.1.3"
-description = """
-GraphQL-core is a Python port of GraphQL.js,
- the JavaScript reference implementation for GraphQL."""
+description = "GraphQL-core is a Python port of GraphQL.js"
 license = "MIT"
 authors = [
     "Christoph Zwerschke <cito@online.de>"
