diff --git a/setup.py b/setup.py
index f0ee2cf..f84e265 100755
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@ def main():
         },
         license='MIT',
         options={"bdist_wheel": {"universal": True}},
-        packages=['CloudFlare', 'CloudFlare/tests', 'cli4', 'examples'],
+        packages=['CloudFlare', 'cli4'],
         test_suite="CloudFlare.tests",
         include_package_data=True,
         data_files=[('share/man/man1', ['cli4/cli4.1'])],
