diff --git a/catalyst/defaults.py b/catalyst/defaults.py
index 705ae931..573575fa 100644
--- a/catalyst/defaults.py
+++ b/catalyst/defaults.py
@@ -33,10 +33,9 @@ valid_config_file_values.extend([ "distcc", "envscript",
 
 # set our base defaults here to keep
 # them in one location.
-BASE_GENTOO_DIR = "/var/gentoo"
-REPODIR = BASE_GENTOO_DIR + "/repos"
-DISTDIR = BASE_GENTOO_DIR + "/distfiles"
-PKGDIR = BASE_GENTOO_DIR + "/packages"
+REPODIR = "/var/db/repos"
+DISTDIR = "/var/cache/distfiles"
+PKGDIR = "/var/cache/binpkgs"
 MAINREPO = "gentoo"
 PORTDIR = REPODIR + "/" + MAINREPO
 
